lib/src/commands/packages/commands/check/commands/licenses.dart

```diff
@@ -21,7 +21,13 @@ class PackagesCheckLicensesCommand extends Command<int> {
     Logger? logger,
     PubLicense? pubLicense,
   })  : _logger = logger ?? Logger(),
-        _pubLicense = pubLicense ?? PubLicense();
+        _pubLicense = pubLicense ?? PubLicense() {
+    argParser.addFlag(
+      'ignore-failures',
```

Comment:
Should we be more verbose here? Something like `--ignore-retrive-failure`. First time that I saw the flag, I initially thought it was about ignoring licenses that aren't permitted.